indy_node/test/upgrade/test_node_control_tool_resolves_dep_top_level.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+from indy_node.utils.node_control_tool import NodeControlTool
+from plenum.test.helper import randomText
+
+
+EXT_PKT_VERSION = '7.88.999'
+EXT_PKT_NAME = 'SomeTopLevelPkt'
+node_package = ('indy-node', '0.0.1')
+PACKAGE_MNG_EXT_PTK_OUTPUT = "Package: {}\nStatus: install ok installed\nPriority: extra\nSection: default\n" \
+                             "Installed-Size: 21\nMaintainer: EXT_PKT_NAME-fond\nArchitecture: amd64\nVersion: {}\n" \
+                             "Depends: {}\nDescription: EXT_PKT_DEPS-desc\n" \
```

Comment:
Please add a couple of other dependencies in addition to indy-node to check that they also taken into account